Sources/Tools/MaxMain/plMaxNode.cpp

```diff
@@ -2075,8 +2075,6 @@ bool plMaxNode::ConvertToOccluder(plErrorMsg* pErrMsg, bool twoSided, bool isHol
     plLocation nodeLoc = GetLocation();
 
     bool moving = IsMovable();
-    if( moving )
-        moving++;
```

Comment:
What in the world 😱

Reply:
Extra truthy...

Reply:
Sometimes the truth just isn't `true` enough